projects/sensory_interface/public/app.js

```diff
@@ -25,4 +30,43 @@ function fillRow(values, currentTr) {
         td.appendChild(document.createTextNode(myValue));
         currentTr.appendChild(td);
     }
+}
+
+function addOnClickSoundToTable() {
+    let element;
+    for (element of document.getElementsByTagName("td")) {
+        element.addEventListener("click", startSoundPlayback);
+    }
+}
+
+function startSoundPlayback() {
+    if (context.state == "suspended") {
+        context.resume();
+    }
+    let request = new XMLHttpRequest();
+    request.open("get", "beep_digital.mp3", true);
+    request.responseType = "arraybuffer";
+    request.onload = function () {
+        let data = request.response;
+        audioRooting(data);
+    };
+    request.send();
+}
+
+function audioRooting(data) {
```

Comment:
Not sure what `audioRooting` except convert `data` to `source` and play it.

How about we merge `playSound` into `audioRooting` and rename it all to `playSound`?

Alternatively, we can keep the 2 methods and rename them to:
`playSoundFromData()`
`playSoundFromBufferSource()`

Reply:
Which option of this 2 you prefer Ofer? I think keeping the "playSound()" function is good because we have also "stopSound()".

Reply:
Sure